examples/Rakefile

```diff
@@ -0,0 +1,37 @@
+require 'fastly_nsq'
+require 'fastly_nsq/rake_task'
+
+##
+# Both topic and channel will need to be passed into the rake task when
+# it is called.
+#
+# ex: `rake listen_task[my_topic,my_channel]`
+# topic will be 'my_topic' and channel will be 'my_channel'
+#
+MessageQueue::RakeTask.new(:listen_task, [:topic, :channel])
```

Comment:
As discussed IRL, make sure this runs well in a Rails app because I recall having to include some kind of `:environment` flag to include the rest of the application code - I got a lot of `Undefined Method BlahClass` but maybe it's not necessary when defining tasks in this way.